src/main.js

```diff
@@ -47,4 +47,15 @@ async function main() {
   createSessionsTable(sessionsList, anchor)
 }
 
-main()
+// Indicator if table has already bean build
```

Comment:
Typo here: `hear` instead of `here` ;)

Reply:
I cannot see "hear" in this comment. Are you talking about "bean" that should be "been" ?